src/drawer/drawer.spec.tsx

```diff
@@ -49,11 +51,12 @@ describe('Drawer', () => {
         </Drawer>
       </React.Fragment>
     );
+    expect(asFragment()).toMatchSnapshot();
   });
 
-  it('can open', (done) => {
-    const el = mount(
-      <Drawer dismissible>
+  it('can open', () => {
+    const { container } = render(
+      <Drawer dismissible open>
```

Comment:
We could remove the "open" attribute and then rerender the drawer with open and verify.